### PR TITLE
yet another outsider buff

### PR DESCRIPTION
**Pull request by benj8560**
## About The Pull Request
Allows outsiders to use nanogates/psionic organs. Also fixes a small issue I noticed basic gates and organs both lacked a blacklist for AI/cyborg. So fixed that to! First PR so make sure I'm not gonna make anything explode. Tested locally and works fine.
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/datums/setup_option/core_implants.dm [hunk 1]
```diff
@@ -91,7 +93,6 @@
 	particular design is an opifex original and one of the best that can be found in the galaxy."
 	implant_organ_type = "opifex nanogate"
 	restricted_jobs = list(
-		/datum/job/outsider, // Nanogates are only available to colonist or allies.
```
Comment (Trilbyspaceclone) [inline code/datums/setup_option/core_implants.dm:94]: I accually agree with this one

Comment (cdb-is-not-good) [reply code/datums/setup_option/core_implants.dm:94]: Aren't opifex the ones who produce nanogates natively and thus should have access to them, in theory

Comment (Trilbyspaceclone) [reply code/datums/setup_option/core_implants.dm:94]: Ye thats why i accually agree on this one

Comment (benj8560) [reply code/datums/setup_option/core_implants.dm:94]: they are disagreeing on the rest. Bird gate = good 
rest = bad
 is the word of the cat.
 
 Personally I don't think its to big a deal. The biggest buff is access to healing something as is outsiders already have with combat medkits and such. Personally I'd likely even take other buffs over the healing with a standard gate given those combat medkits are handy.

**Comment by Trilbyspaceclone** [comment]
Righto to keep things LOOOOORE complaint
I think that the outsiders should get *different* implants then base/church options

Like rather then being able to pick psionic implant itself give them a `stressed` or `overly taxed` verson that has some spells removed like JTN, less regen ect ect


For The church implant make that be like `waning` and have the base churchie spells with some less regen again or lower max cap

**Comment by benj8560** [comment]
I did not allow use of the church implants. Did not speak of it in DMs at all. Felt odd in are lore. I do however....agree it might make for some interesting situations where outsiders of the faith are being harassed by prospecting/shield and church ends up involved.

 The nanogates (aside from birbs duh) not all that impressive again given the fact they have chems at their disposal....but I personally agreed and suggested against psi. Psionics is a tad strong for outsiders already prone to self antaging. And when they can easily just cook up a fresh one if they get PKed or magically have their organ replaced every shift if someone removes it unlike on the colony.

Also gaining stats in a bit of a pain for outsiders and they don't REALLY start with any making the psionic organ rather underwhelming early game.  Extra incoming damage will likely be a hindrance more then anything until mid game.

My take on things however those kinds of changes are a tad out of the scope of this PR. I just moved/copied/deleted some lines my coding knowhow does not go much beyond that.

**Comment by Trilbyspaceclone** [comment]
I do like the bug fixes in this and the idea im not against its just the lore and balance that need a bit of a poke

**Comment by cdb-is-not-good** [comment]
If we're talking outsider balance can we get rid of their chem dispenser.....Grongo suggested giving them just a board and making them scrap for the parts

**Pull request merged**